data/json/itemgroups/Locations_MapExtras/locations_commercial.json

```
@@ -758,6 +759,7 @@
       [ "helmet_liner", 20 ],
       [ "jacket_army", 40 ],
       [ "helmet_army_outdated", 10 ],
+      [ "helmet_soviet_tanker", 10 ],
```

```suggestion
      [ "helmet_soviet_tanker", 1 ],
```
Cut down to 10% of chance of a WW2 US Helmet